src/containers/announcements/index.tsx

```diff
@@ -1,11 +1,15 @@
 import React, { useEffect } from 'react';
 import { Helmet } from 'react-helmet';
 import { Typography } from 'antd';
-import AnnouncementsList from '../../components/announcementsList';
+import AnnouncementsList from '../../components/announcements-listing/AnnouncementsList';
```

Comment:
Why are you changing the naming? I believe camelCase is the way we wanna do it, with an index.tsx file in there to make the imports even easier.